app/features/top-search/SplatoonPlayerRepository.server.ts

```diff
@@ -7,3 +8,72 @@ export function unlinkPlayerByUserId(userId: number) {
 		.where("SplatoonPlayer.userId", "=", userId)
 		.execute();
 }
+
+function selector() {
+	return db
+		.selectFrom("XRankPlacement")
+		.select([
+			"XRankPlacement.id",
+			"XRankPlacement.weaponSplId",
+			"XRankPlacement.name",
+			"XRankPlacement.power",
+			"XRankPlacement.rank",
+			"XRankPlacement.month",
+			"XRankPlacement.year",
+			"XRankPlacement.region",
+			"XRankPlacement.playerId",
+			"XRankPlacement.mode",
+		])
+		.leftJoin("SplatoonPlayer", "XRankPlacement.playerId", "SplatoonPlayer.id")
```

Comment:
```suggestion
		.innerJoin("SplatoonPlayer", "XRankPlacement.playerId", "SplatoonPlayer.id")
```

This used to be a left join but i believe inner is the correct choice here as XRankPlacement should always have corresponding SplatoonPlayer